[Schema] format information mentioned. Open-sourced sdk mentioned as well (#101)
UnumID/UnumID.github.io: docs/schema.mdx

:::

:::important

> Given use of JSON Schema standard for schema definitions, we hope to make <a href="/terminology#customer"><Tip type="customer"/></a> defined credentials schemas a reality very soon. However, until then, all credential types have schemas defined by Verified Inc.. Please let us know if you would like to define a new credential schema for your use case.

Reviewer: ```suggestion
We hope to soon make it possible for you as a <a href="/terminology#customer"><Tip type="customer"/></a> to define your own credential schemas. But **currently, we Verified Inc. define all credential schemas.** If you would like a new credential schema for your use case, please let us know and we will gladly define one for you.
```

                ],
                "type": "string"
            }
        },
        "required": [
            "address"
        ]
    },
    // highlight-end
    "jsonLD": {
        "@type": "AddressCredential",
        "address": "schema:address",
        "@context": "https://schema.unumid.co"
    },
    "attributes": {
        "type": "AddressCredential",
        "attributes": [
            {
                "key": "address",
                "label": "Address",
                "comment": "Physical address.",
                "displayFormat": "Address"
            }
        ]
    }
}
```

You can instead just use the `/jsonSchema` path to only get that information, e.g. https://schema.verified.inc/jsonSchema/FirstNameCredential:
```json title="Example FirstNameCredential JSON Schema"
{
    "$id": "FirstNameCredential",
    "additionalProperties": false,
    "type": "object",
    "properties": {
        "firstName": {
            "description": "A person's first name",
            "examples": [
                "John",
                "Mary Kate"
            ],
            "type": "string"
        }
    },
    "required": [
        "firstName"
    ]
}
```

:::tip

> You will notice that each attribute will have a `description` and `examples` to assist your understanding of the credential's schema. If more detailed information is required our [schema-sdk](https://github.com/UnumID/schema-sdk) is open-sourced and our [formats](https://github.com/UnumID/schema-sdk/blob/main/src/formats.ts) are free to be analyzed.

Reviewer: ```suggestion
Note that each attribute has a `description` and `examples` to help you understand the credential's schema. If you need more detailed information, look at our open source [schema-sdk](https://github.com/UnumID/schema-sdk) and [formats](https://github.com/UnumID/schema-sdk/blob/main/src/formats.ts).
```

:::

## Example

After first grabbing the JSON schemas for the Address, FirstName, and LastName Credentials and understand their attribute's description and examples, we can create compliant credential bodies.

> Below is code level example of those credentials with valid values being constructed for credential issuance. In this example we are following the [AddressCredential](https://schema.verified.inc/jsonSchema/AddressCredential), [FirstNameCredential](https://schema.verified.inc/jsonSchema/FirstNameCredential), and [LastNameCredential](https://schema.verified.inc/jsonSchema/LastNameCredential) JSON schemas.

Reviewer: ```suggestion
Next, we construct valid credential bodies, according to the schemas:
```
